spec/models/booking_response_spec.rb

```diff
@@ -57,6 +57,18 @@
         end
       end
 
+      context 'and the allowed visitor is a child' do
+        let(:visit) { create(:visit) }
+
+        before do
+          visit.visitors.update_all(date_of_birth: 5.years.ago)
```

Comment:
Is it worth making this text explicit regarding the difference between a 'child' and 'under 18'? (i.e. this is a test about under 18, not about children)

I'm not sure if this class is currently `adult_age` aware, but the age in this test should be greater than `adult_age` but less than 18.